build.vsh

```diff
@@ -79,9 +60,7 @@ fn build(mode ReleaseMode, explicit_debug bool) {
 	prepare_output_dir()
 	println('${term.green('✓')} Prepared output directory')
 
-	cmd := mode.cmd()
-	cmd_name := mode.str()
-	println('Building v-analyzer in ${term.bold(cmd_name)} mode...')
+	println('Building v-analyzer in ${term.bold(mode.str())} mode...')
```

Comment:
should not that be: `mode.cmd().str()` ?

Reply:
what does .cmd() do?

Reply:
> should not that be: `mode.cmd().str()` ?

not necessary, since this is just the name of the mode that should be printed.

Before it was:
https://github.com/v-analyzer/v-analyzer/blob/2f24b0377ddb865eb4285634f6945cb4ee3f395c/build.vsh#L36C1-L43C1

This just returned a callback and the function name was used as printed name. 

Instead it can be simpler. Without the callback in one compile function.
https://github.com/ttytm/v-analyzer/blob/5c968846cb0e8b2a21f9d2d88c3ad88ffea071da/build.vsh#L41C1-L48C2

Reply:
let me split this into two PRs. Will be simpler to follow